packages/nextjs/src/server/withClerkMiddleware.ts

```diff
@@ -44,6 +45,10 @@ export const withClerkMiddleware: WithClerkMiddleware = (...args: unknown[]) =>
 
   const proxyUrl = opts?.proxyUrl || PROXY_URL;
 
+  if (!!proxyUrl && !isHttpOrHttps(proxyUrl)) {
+    throw new Error(`Only a absolute URL that starts with https is allowed to be used in SSR`);
```

Comment:
@nikosdouvlis i think we should add a ticket to create `errors.ts` to nextjs, like our other packages (eg remix).